tests/testthat/_snaps/unbump_version/NEWS.md

```diff
@@ -1,4 +1,4 @@
-<!-- NEWS.md is maintained by https://cynkra.github.io/fledge, do not edit -->
+<!-- NEWS.md is maintained by https://fledge.cynkra.com/, do not edit -->
```

Comment:
Where does that change come from? Can we move that to a separate PR?

Reply:
ooooh right I see what you mean now, let me change that.

Reply:
done